gp-templates/project-delete.php

```diff
@@ -7,11 +7,11 @@
  * @since 2.0.0
  */
 
-gp_title( sprintf( __( 'Delete Project %s &lt; GlotPress', 'glotpress' ),  $project->name ) );
+gp_title( sprintf( __( 'Delete Project "%s" &lt; GlotPress', 'glotpress' ),  $project->name ) );
 gp_breadcrumb_project( $project );
 gp_tmpl_header();
 ?>
-<h2><?php printf( __( 'Delete project &#8220;%s&#8221;', 'glotpress' ), esc_html( $project->name ) ); ?></h2>
+<h2><?php printf( __( 'Delete Project "%s"', 'glotpress' ), esc_html( $project->name ) ); ?></h2>
```

Comment:
Same as above.

Reply:
Ok @toolstack 
I'll review all of those with lowercase, keeping consistency between them

Reply:
@toolstack this particular case isn't a button, is a heading.
Should it be kept consistent with the button, as you've mentioned?

Reply:
We're a little inconsistent on that point in the code, I'd say lower case because while it is a heading, which is often capitalizes the first letter of a word, in this case it's more of a sentence so you wouldn't.

I'd say don't capitalize in these cases.

Reply:
Ok, so it's done.